src/Manavo/BootstrapForms/FormBuilder.php

```diff
@@ -203,13 +202,14 @@ protected function checkable($type, $name, $value, $checked, $options)
     public function checkbox(
         $name,
         $value = 1,
-        $label = null,
         $checked = null,
         $options = []
     ) {
         $checkable = parent::checkbox($name, $value, $checked, $options);
 
-        return $this->wrapCheckable($label, 'checkbox', $checkable);
+        return @$options['label'] ?
```

Comment:
'empty()` or `isset()`? I could see not outputting label markup when `['label' => '']` being confusing and hard to debug. Better to output the label whenever the user sets the `label` option.